### ISSUE TITLE
Support remove vertex / edge from a loaded graph.

### ISSUE THREAD

**Issue by siyuan0322** [alibaba/GraphScope#196]
**Is your feature request related to a problem? Please describe. **
GraphScope support a constraint removal of vertices and edges after #193 is merged, which means one can only remove items that are not been loaded into vineyard yet.

However, we may want to support fully functional graph manipulation.

**Describe the solution you'd like**
Probably we could utilize the generic project operator to simulate removal. #134 

**Additional context**
Low priority.
